packages/theme-material-ui/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@hig/theme-material-ui",
+  "version": "1.0.0-alpha",
+  "description": "HIG Theme Material UI",
+  "author": "Autodesk Inc.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Autodesk/hig.git"
+  },
+  "files": [
+    "build/*"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "hig-scripts-test",
+    "build": "yarn run build:themes",
+    "build:core": "hig-scripts-build",
+    "build:themes": "babel-node scripts/buildThemes",
+    "lint": "hig-scripts-lint",
+    "release": "hig-scripts-release"
+  },
+  "peerDependencies": {
+    "@hig/theme-context": "^3.0.0",
```

Comment:
i don't think you need `@hig/theme-context` as part of this package, as I didn't see it used. also `@hig/theme-data` should be set to the latest version.